net/davix/src/TDavixFile.cxx

```diff
@@ -417,6 +417,17 @@ void TDavixFileInternal::parseConfig()
    if (gDebug > 0)
       Info("parseConfig", "Setting CAcheck to %s", ((ca_check_local) ? ("true") : ("false")));
 
+   std::string prefix = "Bearer ";
+   // setup Bearer token auth
+   env_var = gEnv->GetValue("Davix.HTTP.Token", getenv("HTTP_TOKEN"));
+      if (gDebug > 0)
```

Comment:
This `if` statement can be removed I think.  Logging is covered inside the `if (env_var)` block.